aspnetcore/choose-aspnet-framework.md

```diff
@@ -19,28 +19,28 @@ No matter the web application you are creating, ASP.NET has a solution for you:
 
 ## ASP.NET Core
 
-ASP.NET Core is a new open-source and cross-platform .NET framework for building modern cloud-based web applications on Windows, Mac, or Linux.
+ASP.NET Core is a new, open-source, cross-platform framework for .NET to build modern, cloud-based web applications on Windows, macOS, or Linux.
```

Comment:
Since we're approaching 2.0 I think we can drop "new"

I prefer "for building" over "to build" in this sentence -- cross-platform framework for building [can omit ".NET" here]